build.gradle

```diff
@@ -11,23 +11,25 @@ buildscript {
         classpath "org.ajoberstar:gradle-git:1.7.2"
     }
 }
-description = 'swagger2markup Build'
-version = '1.3.4-SNAPSHOT'
-ext.releaseVersion = '1.3.3'
```

Comment:
This is property is used inside of the ascidoc documentation. It should not be deleted.

Reply:
Often I want to update the documentation, but the release version did not change. I don't want that the SNAPSHOT version is mentioned in the documentation as the last released version.